cmd/observe/flows.go

```diff
@@ -316,13 +316,13 @@ more.`,
 
 	filterFlags.Var(filterVar(
 		"from-pod", ofilter,
-		"Show all flows originating in the given pod name ([namespace/]<pod-name>). If namespace is not provided, 'default' is used"))
+		"Show all flows originating in the given pod name prefix([namespace/]<pod-name>). If namespace is not provided, 'default' is used"))
 	filterFlags.Var(filterVar(
 		"pod", ofilter,
-		"Show all flows related to the given pod name ([namespace/]<pod-name>). If namespace is not provided, 'default' is used"))
+		"Show all flows originating in the given pod name prefix ([namespace/]<pod-name>). If namespace is not provided, 'default' is used."))
```

Comment:
I think `related to` was correct here as this flag is about flows that are either from or to the given pod.
```suggestion
		"Show all flows related to the given pod name prefix ([namespace/]<pod-name>). If namespace is not provided, 'default' is used."))
```